test/Serilog.Tests/Debugging/SelfLogTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Serilog.Debugging;
+using Xunit;
+
+namespace Serilog.Tests.Debugging
+{
+    public class SelfLogTests
+    {
+        [ThreadStatic]
+        static List<string> Messages;
+        
+        [Fact]
+        public void MessagesAreWrittenWhenOutputIsSet()
+        {
+            Messages = new List<string>();
+            SelfLog.Enable(m =>
+            {
+                Messages = Messages ?? new List<string>();
```

Comment:
There are some references to `messages` vs `events`.  Should these and the test case use `event` for clarity?